randovania/games/prime3/logic_database/Pirate Homeworld - Research.txt

```diff
@@ -221,18 +221,50 @@ Extra - asset_id: 3594029436312475339
       After Proving Grounds Barrier Removed
   > Event - Gandrayda
       Any of the following:
-          Combat (Expert)
           All of the following:
-              Morph Ball
+              Grapple Lasso and Fight Ghor-G
               Any of the following:
-                  Grapple Lasso and X-Ray Visor
-                  Combat (Beginner) and Enter Hypermode
+                  # Fighting Gandrayda
+                  Combat (Expert)
+                  All of the following:
+                      # Fight with Hypermode
+                      Enter Hypermode
+                      Any of the following:
+                          Combat (Intermediate) or Damage ≥ 399
+                          Combat (Beginner) and Damage ≥ 199
+                  Any of the following:
+                      # Fight Without Hypermode
+                      Damage ≥ 599
+                      Combat (Beginner) and Damage ≥ 299
+                      Combat (Intermediate) and Damage ≥ 199
+                      Combat (Advanced) and Damage ≥ 99
               Any of the following:
-                  Morph Ball Bomb or Boost Ball
-                  Hyper Ball and Knowledge (Beginner) and Enter Hypermode
-          All of the following:
-              Combat (Advanced)
-              Boost Ball or Use Screw Attack (Space Jump)
+                  # Fighting/Avoiding Invisible Phase
+                  Combat (Intermediate) or Invisible Objects (Intermediate)
+                  Combat (Beginner) and Damage ≥ 49
+          Any of the following:
```

Comment:
This `Any` seems unnecessary. I see why you did it because of the comment, but you could just list each alternative up front and just extend each comment like `Skip Phases with Screw Attack`, etc.